### ISSUE TITLE
Training error

### ISSUE THREAD

**Issue by Haithienbao** [XingangPan/SCNN#126]
When i try to train this code, i get error like this:
![image](https://user-images.githubusercontent.com/81343128/112602986-f093b900-8e46-11eb-81fd-f5a8ec7b19b0.png)
data created	
data loaded	
data created	
data loaded	
init trainer	
training	
=> Training epoch # 1	
/torch/install/bin/luajit: /torch/install/share/lua/5.1/threads/threads.lua:183: [thread 2 callback] ./datasets/lane.lua:22: attempt to index local 'label' (a number value)
stack traceback:
	./datasets/lane.lua:22: in function 'get'
	./dataloader.lua:96: in function <./dataloader.lua:92>
	[C]: in function 'xpcall'
	/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
	/torch/install/share/lua/5.1/threads/queue.lua:65: in function </torch/install/share/lua/5.1/threads/queue.lua:41>
	[C]: in function 'pcall'
	/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:15: in main chunk
stack traceback:
	[C]: in function 'error'
	/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
	./dataloader.lua:157: in function '(for generator)'
	./train.lua:58: in function 'train'
	main.lua:51: in main chunk
	[C]: in function 'dofile'
	/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x55c09631d570
load image failed!
load image failed!

I run this code in google colab, cuda10,
Could you have me solve this problem?
Thank you!
